pages/Recommendation.html

```diff
@@ -65,8 +65,14 @@
           data-auto-select-fields-to-include="true"></div>
         <div class="CoveoResultList" data-layout="card" data-wait-animation="fade"
           data-auto-select-fields-to-include="true"></div>
-        <div class="CoveoPager"></div>
-        <div class="CoveoResultsPerPage"></div>
+        <div class="coveo-results-footer">
+          <div class="coveo-results-footer-left">
+            <div class="CoveoPager"></div>
+          </div>
+          <div class="coveo-results-footer-right">
+            <div class="CoveoResultsPerPage"></div>
+          </div>
+        </div>
```

Comment:
This is the only page that has a different results footer in comparison to the others; it doesn't have a logo.